### PR TITLE
[ROX-11295] Postgres store for nodes

### PR DESCRIPTION
**Pull request by md2119**
## Description

- Added Postgres store for nodes
- Updated node merge code to account for suppressed state
- Updated split/merge code with deduplication
- Removed newly added `storage.NodeComponentCVEEdge` proto

## Checklist
- [ ] Investigated and inspected CI test results
- [X] Unit test and regression tests added
- ~Evaluated and added CHANGELOG entry if required~
- ~Determined and documented upgrade steps~
- ~Documented user facing changes~

If any of these don't apply, please comment below.

## Testing Performed
Unit

### CODE REVIEW

File: pkg/postgres/schema/node_components_cves_edges.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+)
+
+var (
+	// CreateTableNodeComponentsCvesEdgesStmt holds the create statement for table `node_components_cves_edges`.
+	CreateTableNodeComponentsCvesEdgesStmt = &postgres.CreateStmts{
+		Table: `
+               create table if not exists node_components_cves_edges (
+                   Id varchar,
+                   IsFixable bool,
+                   FixedBy varchar,
```
Comment (connorgorman) [inline pkg/postgres/schema/node_components_cves_edges.go:23]: Can we fix this redundancy? FixedBy != null is IsFixable

File: central/nodecomponentcveedge/datastore/store/postgres/gen.go [hunk 2]
```diff
@@ -0,0 +1,3 @@
+package postgres
+
+//go:generate pg-table-bindings-wrapper --type=storage.ComponentCVEEdge --table=node_components_cves_edges --search-category NODE_COMPONENT_CVE_EDGE --references=node_components:storage.ImageComponent,node_cves:storage.CVE --join-table true
```
Comment (connorgorman) [inline central/nodecomponentcveedge/datastore/store/postgres/gen.go:3]: Why still storage.ImageComponent, and storage.CVE? I was under the impression we were going to split these out?

File: pkg/postgres/schema/node_components_cves_edges.go (hunk 1 above)
Comment (md2119) [reply pkg/postgres/schema/node_components_cves_edges.go:23]: Such tasks should not go under postgres umbrella. It is more of a deprecation that requires us to visit UI and service layer, wrapper code, etc.

File: proto/storage/cve.proto [hunk 3]
```diff
@@ -88,6 +88,44 @@ message CVE {
     VulnerabilitySeverity       severity            = 20 [(gogoproto.moretags) = "search:\"Severity\""];
 }
 
+message CVEInfo {
+    message Reference {
```
Comment (connorgorman) [inline proto/storage/cve.proto:92]: I guess we should keep this, but I think we can just deprecate it. It's never been widely used in our platform

**Review by connorgorman — APPROVED** [review]

**Pull request merged**